### DISCUSSION TITLE
VS errors - command not found

### DISCUSSION THREAD

**Discussion by markharrison** [community/community#7005] [Copilot Conversations] [answered]
<img width="448" alt="Untitled" src="https://user-images.githubusercontent.com/5177824/139445753-62a566a5-5da6-413d-addc-453801f86702.png">

I have tried to authenticate to the Tech Preview with my wrong GH account.  How do I reset things ?   I did delete my .vscode folder and then reinstalled CoPilot extension - but get these errors ?

**Comment by markharrison** [comment, marked as answer]
solved. Installed the GH extension and was able to logon with correct user.

**Comment by heyitsranjan** [comment]
Same issue from the beginning in my Windows Machine. Tried re-installing  Github Copilot Extension, also removed VS Code cache. Still giving same error

**Reply by davecheney** [reply]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by heyitsranjan** [reply]
Yes. I'm trying on my office laptop. Tried with VPN turned off, still getting the same issue.

**Reply by davecheney** [reply]
@devsmranjan I see no connections to our server from your token. I looks like something on your network or in your laptop is intercepting HTTPS connections.
